helm/infinibox-csi-driver/templates/storageclass-fc.yaml

```diff
@@ -0,0 +1,13 @@
+apiVersion: storage.k8s.io/v1
```

Comment:
As per discussion, we need to remove create StorageClass ymal from helm deployment